priv/templates/nimble_template/Dockerfile.eex

```diff
@@ -1,52 +1,55 @@
-FROM <%= docker_build_base_image %> AS build
+ARG ELIXIR_IMAGE_VERSION=<%= elixir_version %>
+ARG ERLANG_IMAGE_VERSION=<%= erlang_version %>
+ARG RELEASE_IMAGE_VERSION=<%= alpine_version %>
+
+FROM hexpm/elixir:${ELIXIR_IMAGE_VERSION}-erlang-${ERLANG_IMAGE_VERSION}-alpine-${RELEASE_IMAGE_VERSION} AS build
+
+RUN apk update && \
+    apk upgrade --no-cache && \
+    apk add --no-cache \
+      nodejs \
+      npm \
+      git \
+      build-base && \
+    mix local.rebar --force && \
+    mix local.hex --force
 
-# install build dependencies
-RUN apk add --no-cache build-base npm git && \
-    mix local.hex --force && \
-    mix local.rebar --force
-
-# prepare build dir
 WORKDIR /app
 
-# set build ENV
+COPY . .
+
 ENV MIX_ENV=prod
 
-# install mix dependencies
-COPY mix.exs mix.lock ./
-COPY config config
-RUN mix do deps.get, deps.compile
+RUN mix do deps.get, deps.compile, compile
 <%= if web_project? do %>
-# build assets
-COPY assets/package.json assets/package-lock.json ./assets/
-RUN npm --prefix ./assets ci --progress=false --no-audit --loglevel=error
-
-COPY priv priv
-COPY assets assets
-RUN npm run --prefix ./assets deploy
-RUN mix phx.digest
+RUN cd assets && \
+		npm ci --progress=false --no-audit --loglevel=error && \
```

Comment:
hi @andyduong1920 !

I have a question here, I have a problem with this line when I try to build the Docker image, 
In `assets`, I used `yarn` to install then dependencies, then it will fail here right? Because there is no `npm-package.lock` if you use `yarn`.
So we have to use `npm` to install the packages, is it correct?

Reply:
Yup correct @byhbt, the Phoenix generator only support npm (package-lock.json) for now. ref https://hexdocs.pm/phoenix/up_and_running.html#content